apps/chat/src/components/Chat/ChatHeader.tsx

```diff
@@ -283,7 +287,7 @@ export const ChatHeader = ({
                 </button>
               </Tooltip>
             )}
-            {isPlayback && (
+            {isPlayback && !isExternal && !conversation.publishedWithMe && (
```

Comment:
Why do we need last condition?
```suggestion
            {isPlayback && !isExternal && (
```

Reply:
Changed as suggested.
Thank you!